package.json

```diff
@@ -136,7 +136,7 @@
     "tape-promise": "4.0.0",
     "ts-jest": "27.0.3",
     "ts-loader": "9.2.5",
-    "ts-node": "10.2.0",
+    "ts-node": "10.7.0",
```

Comment:
@Leeyoungone I remember this being done in another pull request that's unrelated to this one so you might want to mark this as a dependent of that PR if it otherwise doesn't work.

Reply:
ooo neat neat! thank you, I think I might have changed the `ts-node`without meaning to so I'll change it back